Harvest.Web/ClientApp/src/Shared/ShowFor.tsx

```diff
@@ -2,25 +2,26 @@ import React, { useContext, useMemo } from "react";
 
 import AppContext from "./AppContext";
 import { RoleName } from "../types";
+import { isBoolean, isFunction } from "../Util/TypeChecks";
 
 interface Props {
   children: any;
   roles: RoleName[];
+  condition?: boolean | (() => boolean);
 }
 
 export const ShowFor = (props: Props) => {
   const { children, roles } = props;
   const userRoles = useContext(AppContext).user.roles;
+  const conditionSatisfied = isBoolean(props.condition) ? props.condition : isFunction(props.condition) ? props.condition() : true;
```

Comment:
either write out or throw some parenthesis in there.  i don't like the visually ambiguous double-ternary operator.

Reply:
Fixed